src/lib/calculation.ts

```diff
@@ -27,7 +27,8 @@ export function calculate(input: TaxFormUserInput): TaxForm {
     r005_meno: firstName,
     r007_ulica: input.r007_ulica,
     r008_cislo: input.r008_cislo,
-    r009_psc: input.r009_psc,
+    psc: input.psc,
```

Comment:
`psc` tam este musi byt ak je tam `r009_psc`?

Reply:
Pre vyplnenie DP nepotrebujeme `psc` (formatovane s medzerou), staci nam `r009_psc`. Ale myslim ze ty si mi hovoril ze `TaxForm` ma byt superset `TaxFormUserInput`-u. Tak neviem, porad :)

Reply:
No tiez sa v tom uz stracam, mozno ta design decision uz nedava zmysel :D Na zaciatku to tak bolo ze v `TaxFormUserInput` nebolo nic co nie je v `TaxForm`. Teraz sa zacinaju kopit fieldy co su len v inpute a nie v TaxForm. Teraz mi uz dava vacsi zmysel tieto dva interfacy oddelit, aj za cenu nejakej duplicity medzi nimi.

Reply:
Dalsi argument pre oddelenie je, ze v inpute su aj numericke riadky stringy a v taxForm su uz numbers.

Reply:
Takze ok, teraz to moze byt takto (asi ti TS dava chyby ked tam nie su obidvoje `psc`), ale mali by sme to asi zrefaktorovat a oddelit niekedy.